src/osd/osd_types.h

```diff
@@ -4301,4 +4301,31 @@ struct PromoteCounter {
   }
 };
 
+/** store_statfs_t
++* ObjectStore full statfs information
++*/
+struct store_statfs_t
+{
+  uint64_t available = 0;              // Free blocks available
+
+  uint64_t blocks = 0;                 // Total data blocks
+  uint32_t bsize = 0;                  // Optimal transfer block size
+
+  int64_t allocated = 0;               // Bytes allocated by the store
+  int64_t stored = 0;                  // Bytes actually stored by the user
+  int64_t compressed = 0;              // Bytes stored after compression
+  int64_t compressed_allocated = 0;    // Bytes allocated for compressed data
+  int64_t compressed_original = 0;     // Bytes that were successfully compressed
+
+  int64_t files = 0;                   // Total file nodes in filesystem, preserved for backward compatibility!
+  int64_t ffree = 0;                   // Free file nodes in filesystem, preserved for backward compatibility!
```

Comment:
since we don't actually fill these with correct info, let's drop them.  if a caller needs them we can fabricate a fictitious value there.. but hopefully they don't!